tests/sdk/v2/driver/regression/data/v0.47.0_mixed.json

```diff
@@ -0,0 +1 @@
+{"results": [{"value": "100", "serialization_format": "JSON"}, {"chunks": ["gASVEgAAAAAAAACMDnBpY2tsZWRfc3RyaW5nlC4=\n"], "serialization_format": "ENCODED_PICKLE"}], "type": "ComputeEngineWorkflowResult"}
```

Comment:
**non-blocking,thought**: we don't know yet if the new serialization is gonna land in the v0.47.0 release, because we haven't made it yet. If it doesn't we'll have to remember to change the file names accordingly